Correction of Chinese translation &fix the gibberish problem.

MephistoSatanDevil opened the pull request.

MephistoSatanDevil committed:
fix typo & the gibberish problem.

MephistoSatanDevil committed:
fix typo & the gibberish problem.

MephistoSatanDevil committed:
fix typo & the gibberish problem.

MephistoSatanDevil committed:
fix typo & the gibberish problem.

Roberciiik commented on bg1npc/bg1npc.tp2:
```diff
@@ -22,6 +22,25 @@ README ~bg1npc/readme-bg1npc-%LANGUAGE%.html~ ~bg1npc/readme-bg1npc.html~
+
+  ACTION_DEFINE_ARRAY fl#noconvert BEGIN setup-osx setup-unix setup-win32 END
+  ACTION_DEFINE_ARRAY fl#reload BEGIN bg1npc END
+  LAF HANDLE_CHARSETS
```
This function is already invoked by `bg1npc/lib/bg1npc_always.tpa` script so please extend this one instead of calling the function twice.

MephistoSatanDevil replied:
OK, I've withdrawn this change of the bg1npc.tp2 file, though I'm not so sure about the bg1npc_always.tpa change. 
But no matter what changes are made, the console  during installation or the weidu.log file after install always be garbled. So just use the English setup.tra for now.

MephistoSatanDevil replied:
> This function is already invoked by `bg1npc/lib/bg1npc_always.tpa` script so please extend this one instead of calling the function twice.

My fault. have been unchanged.
But now, the console during reinstallation or the weidu.log file after install always be garbled. how to fix it?

MephistoSatanDevil replied:
> This function is already invoked by `bg1npc/lib/bg1npc_always.tpa` script so please extend this one instead of calling the function twice.

My fault. have been unchanged.

And now it test passed. everything looks fine.

MephistoSatanDevil committed:
Cancel changes with bg1npc.tp2

MephistoSatanDevil committed:
covert setup_win32.tra to utf8

MephistoSatanDevil committed:
convert setup-win32.tra to cp936

MephistoSatanDevil committed:
add nocovert filename

MephistoSatanDevil committed:
Delete redundant symbols

MephistoSatanDevil committed:
Fix typo

MephistoSatanDevil committed:
Delete redundant symbols

MephistoSatanDevil committed:
Update for WeiDU-Windows-249-amd64

yoshimo0417 committed:
fix few chinese strings

MephistoSatanDevil committed:
Merge pull request #1 from yoshimo0417/master

fix few chinese strings

MephistoSatanDevil committed:
typo

MephistoSatanDevil committed:
modfy based on the following discussions

https://github.com/The-Gate-Project/DarkHorizons/pull/23

MephistoSatanDevil committed:
Partial corrections of some personal and place names translations.

MephistoSatanDevil committed:
modfy based on the following discussions

https://github.com/The-Gate-Project/DarkHorizons/pull/23

Roberciiik commented on bg1npc/bg1npc.tp2:
```diff
@@ -34,7 +34,8 @@ LANGUAGE ~Polski (Tlumaczenie przez Children of Bhaal)~ ~polish~ ~bg1npc/tra/pol
 LANGUAGE ~Italian (by Coland)~ ~italian~ ~bg1npc/tra/italian/setup.tra~ 
-LANGUAGE ~Chinese~ ~schinese~ ~bg1npc/tra/schinese/setup.tra~
+LANGUAGE ~Simplified Chinese~ ~schinese~ ~bg1npc/tra/english/setup.tra~
+                                         ~bg1npc/tra/schinese/install.tra~
```
@MephistoSatanDevil Please rename the new install.tra file to setup.tra, as the original setup file contains only commands prompt too, then it is the exactly what you need. Simply keep it in UTF-8 and add it to the exclude list.

MephistoSatanDevil replied:
When we simply rename the install.tra file back to setup.tra and keep it in UTF-8 encoding, an error occurs during installation. 
Could you please advise on how to add it to the charset convertion exclude list?

Roberciiik replied:
That's strange that you have some error during installation. Let me try this and I will come back to you. However I meant exclude list as the variable `x#noconvert` that you've already set in this change:

Roberciiik replied:
I opened a PR in your repository fork. I was able to install the mod successfully. If you still see the error, please post a error message or screenshots.

MephistoSatanDevil replied:
> I opened a PR in your repository fork. I was able to install the mod successfully. If you still see the error, please post a error message or screenshots.
I see now. 
Because I was afraid it would affect other languages, so  I did not modify the variable x#reload and x#noconvert before. 

So, by making this change, do the setup.tra files for other languages need to be converted to UTF-8 encoding?

Roberciiik replied:
> Because I was afraid it would affect other languages, so I did not modify the variable x#reload and x#noconvert before.
> So, by making this change, do the setup.tra files for other languages need to be converted to UTF-8 encoding?

This is correct. It is affecting the other languages as well. Still it is better to update those files instead of duplicating the solution and have the same in two files (install and setup) with the same purpose. I will update required files for other languages.

MephistoSatanDevil replied:
> This is correct. It is affecting the other languages as well. Still it is better to update 

those files instead of duplicating the solution and have the same in two files (install and setup) with the same purpose. I will update required files for other languages.

Thank you.

Roberciiik replied:
PR is ready to be merged in your repo fork.

Roberciiik committed:
setup.tra in UTF-8

MephistoSatanDevil committed:
Merge pull request #2 from Roberciiik/master

setup.tra in UTF-8

Roberciiik committed:
ITA & PL setup in correct encoding as well.

MephistoSatanDevil committed:
Merge pull request #3 from Roberciiik/master

ITA & PL setup in correct encoding as well.

yoshimo0417 committed:
Updated some Chinese terms

MephistoSatanDevil committed:
Merge pull request #4 from yoshimo0417/master

Updated some Chinese terms

yoshimo0417 committed:
Updated some Chinese terms

MephistoSatanDevil committed:
Merge pull request #5 from yoshimo0417/master

Updated some Chinese terms

MephistoSatanDevil committed:
Modify the translation of Crossbow in Chinese

MephistoSatanDevil committed:
uodate chinese  translation

MephistoSatanDevil committed:
uodate chinese translation

MephistoSatanDevil committed:
fix chinese translation

MephistoSatanDevil committed:
fix chinese translation

MephistoSatanDevil committed:
fix chinese translation

MephistoSatanDevil committed:
Chinese translation correction.

Corrected some of the translation tone.
Added Chinese translations for prompts.

yoshimo0417 committed:
update chinese translation

MephistoSatanDevil committed:
Merge pull request #6 from yoshimo0417/master

update chinese translation

MephistoSatanDevil committed:
remove prompts.tra

MephistoSatanDevil committed:
remove prompts.tra

MephistoSatanDevil committed:
Merge branch 'Gibberlings3:master' into master

yoshimo0417 committed:
Update X#KISEQU.TRA

MephistoSatanDevil committed:
Merge pull request #7 from yoshimo0417/master

Update X#KISEQU.TRA

MephistoSatanDevil committed:
typo

yoshimo0417 committed:
Correct typo for Simplified Chinese

MephistoSatanDevil committed:
Merge pull request #8 from yoshimo0417/master

Correct typo for Simplified Chinese

MephistoSatanDevil committed:
Merge branch 'Gibberlings3:master' into master
